images/macos/software-report/SoftwareReport.Xcode.psm1

```diff
@@ -220,6 +220,40 @@ function Build-XcodeSimulatorsTable {
     }
 }
 
+function Build-XcodeCommandLineToolsSection {
+    $PKG_PATH = "/Library/Developer/CommandLineTools"
+
+    $clang_version = (&"${PKG_PATH}/usr/bin/clang" --version).Split(' ')[3]
+
+    $clang_version_parts = $clang_version.Split('.')
+    $clang_version_no = ($clang_version_parts[0] -as [int]) * 10 + ($clang_version_parts[1] -as [int])
+
+    $xcode_version = switch ($clang_version_no) {
+        0 {"dunno"}
+        60 {"6.0"}
+        61 {"6.1"}
+        70 {"7.0"}
+        73 {"7.3"}
+        80 {"8.0"}
+        81 {"8.3"}
+        90 {"9.2"}
+        91 {"9.4"}
+        100 {"10.3"}
+        110 {"11.5"}
+        default   {"12.0"}
+    }
+
+    $versions = @(
+        "clang version $clang_version",
+        "Corresponding XCode version $xcode_version"
```

Comment:
What does it mean `Corresponding XCode version`?
Command Line Tools is not related to Xcode. They have similar versioning but it is a separate tool - replacement of Xcode.

I suggest the following